src/protocol/templates/Ping1D.py.in

```diff
@@ -83,7 +82,7 @@ class Ping1D(object):
         return None
 
     ## Handle an incoming messge from the device
-    # Extract message fields into self attributes
+    #  Extract message fields into self attributes
```

Comment:
This message returns something.

Reply:
you are mistaken

Reply:
added @return for __repr__ below

Reply:
Yep, sorry, I did not click in "expand code".

Reply:
I saw it ;)